src/control/Control.Layers.js

```diff
@@ -163,7 +163,7 @@ L.Control.Layers = L.Control.extend({
 		container.setAttribute('aria-haspopup', true);
 
 		L.DomEvent.disableClickPropagation(container);
-		if (!L.Browser.touch) {
+		if ((L.Browser.chrome && L.Browser.pointer) || !L.Browser.touch) {

```

Comment:
Maybe I'm missing something here, but why do we have this conditional at all? Why should scroll events propagate for mobile/touch but not for desktop/non-touch?